### DISCUSSION TITLE
One question about weight factor

### DISCUSSION THREAD

**Discussion by liuyii7** [neperfepx/neper#955] [Q&A] [answered]
Hi, I want to consult one question about weight factor in the generated tessellation file: I noticed sometimes the weight = 0, I want to consult the reason. Many thanks!

**Comment by rquey** [comment, marked as answer]
Weights can be zero, as in the case of a Voronoi tessellation... There is no specific reason for the weights to be zero or non-zero.  If the tessellation is obtained by optimization (as in `-morpho gg`), it is simply that the solution was obtained by setting these specific weights to zero...

**Comment by liuyii7** [comment]
Okay I see, thanks so much for the clarification!

发送自 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Romain Quey ***@***.***>
发送时间: Thursday, December 19, 2024 6:06:49 PM
收件人: neperfepx/neper ***@***.***>
抄送: liuyii7 ***@***.***>; Author ***@***.***>
主题: Re: [neperfepx/neper] One question about weight factor (Discussion #955)


Weights can be zero, as in the case of a Voronoi tessellation... There is no specific reason for the weights to be zero or non-zero. If the tessellation is obtained by optimization (as in -morpho gg), it is simply that the solution was obtained by setting these specific weights to zero...

―
